python/seldon_core/seldon_methods.py

```diff
@@ -78,16 +78,18 @@ def send_feedback(user_model: Any, request: prediction_pb2.Feedback,
     else:
         try:
             return user_model.send_feedback_raw(request)
-        except (NotImplementedError, AttributeError):
```

Comment:
How do AttributeErrors can handled now? Can we add a test

Reply:
Do we need to rescue `AttributeError`? `send_feedback_raw` originally raises `NotImplementedError`.

Reply:
I got your point. I'll add `has_attr` check. Can I also include fix for #700?

Reply:
Yes I added Fix in your top level comment. So yes if you could extend to cover #700